tasks/Debian.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Add apt-key
-  apt_key: id=DF7D54CBE56151BF keyserver=keyserver.ubuntu.com state=present
+  apt_key: id=E56151BF keyserver=keyserver.ubuntu.com state=present
```

Comment:
Its form the official mesosphere site
https://open.mesosphere.com/getting-started/install/